File: naTanjir/UI/natanjir_mobile/lib/models/lokacija.dart
import 'package:json_annotation/json_annotation.dart';
part 'lokacija.g.dart';

@JsonSerializable()
class Lokacija {
  int? lokacijaId;
  int? korisnikId;
  String? adresa;
  double? geografskaDuzina;
  double? geografskaSirina;
  Lokacija();

  factory Lokacija.fromJson(Map<String, dynamic> json) =>
      _$LokacijaFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$LokacijaToJson(this);
}

File: naTanjir/naTanjir.Model/Lokacija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace naTanjir.Model
{
    public class Lokacija
    {
        public int LokacijaId { get; set; }
        public string? Adresa { get; set; }
        public int? KorisnikId { get; set; }
        public decimal GeografskaDuzina { get; set; }

        public decimal GeografskaSirina { get; set; }

    }
}
